Makefile

```diff
@@ -11,7 +11,10 @@ TIMEOUT := 60
 UNION := functional docker crio docker-compose network netmon docker-stability oci openshift kubernetes swarm vm-factory entropy ramdisk shimv2
 
 # skipped test suites for docker integration tests
-SKIP := $(shell bash -f .ci/hypervisors/$(KATA_HYPERVISOR)/filter_docker_$(KATA_HYPERVISOR).sh)
+FILE = .ci/hypervisors/$(KATA_HYPERVISOR)/filter_docker_$(KATA_HYPERVISOR).sh
```

Comment:
`FILE` is a very generic name, can you change it to `FILTER_FILE` or something like that?

Reply:
@devimc changes applied

Reply:
Thanks @GabyCT